src/Components/ADT3DSceneBuilder/Internal/Shared/PanelFooter.styles.ts

```diff
@@ -19,6 +19,6 @@ export const PanelFooterStyles = mergeStyleSets({
 export const PanelFooterStackStyles: IStackStyles = {
     root: {
         display: 'flex',
-        padding: `12px 4px ${BUILDER_FOOTER_BOTTOM_OFFSET}px 4px` // separator has weird built in padding on top
+        padding: `12px 16px ${BUILDER_FOOTER_BOTTOM_OFFSET}px 16px` // separator has weird built in padding on top,
```

Comment:
why did this change? They now seem like they don't match
![image](https://user-images.githubusercontent.com/57726991/172723248-7630e785-afda-4577-86af-c8b7910e0c0f.png)

Reply:
This is from `main`

![image](https://user-images.githubusercontent.com/18181049/172728147-cb9bef39-c9b2-4651-89fd-a368a7d6897f.png)

This change only updates the left & right padding in the `padding: top right bottom left` syntax

Reply:
ahhh fair enough. probalby the difference in the 12 & 16 there then.